src/multirust-cli/setup_mode.rs

```diff
@@ -21,13 +21,20 @@ pub fn main() -> Result<()> {
              .help("Enable verbose output"))
         .arg(Arg::with_name("no-prompt")
              .short("y")
-             .help("Disable confirmation prompt."));
+             .help("Disable confirmation prompt."))
+        .arg(Arg::with_name("default-toolchain")
+             .short("d")
+             .long("default-toolchain")
+             .takes_value(true)
+             .possible_values(&["stable", "beta", "nightly"])
```

Comment:
Yeah I went with those three to start, but this could (should?) be extended to cover any build number in the future